roles/common/tasks/main.yml

```diff
@@ -26,12 +26,18 @@
     gcloud_res_grps: "{{ output.output.data.resource_groups | selectattr('res_group_type', 'equalto', 'gcloud') | list }}"
 
 - name: "filtering the output for duffy resource groups"
-  set_fact: 
+  set_fact:
     duffy_res_grps: "{{ output.output.data.resource_groups | selectattr('res_group_type', 'equalto', 'duffy') | list }}"
 
+#- name: "DEBUG:: checking registered output of os_res_grps"
+#  debug:
+#    msg: "{{ item }}"
+#  with_items: "{{ output.output.data }}"
```

Comment:
I don't disagree. However, these kinds of artifacts already existed in the code. It is a good idea to clean them up.